lemminx-maven/src/test/java/org/eclipse/lemminx/maven/test/PathsTest.java

```diff
@@ -0,0 +1,91 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Red Hat Inc. and others.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.eclipse.lemminx.maven.test;
+
+import static org.eclipse.lemminx.maven.test.MavenLemminxTestsUtils.createDOMDocument;
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import org.eclipse.lemminx.dom.DOMDocument;
+import org.eclipse.lemminx.services.XMLLanguageService;
+import org.eclipse.lemminx.settings.SharedSettings;
+import org.eclipse.lsp4j.CompletionItem;
+import org.eclipse.lsp4j.Position;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TestRule;
+
+public class PathsTest {
+
+	public @Rule TestRule noCentralIndexRule = new NoMavenCentralIndexTestRule();
+	
+	private XMLLanguageService languageService;
+
+	@Before
+	public void setUp() throws IOException {
+		languageService = new XMLLanguageService();
+	}
+
+	@After
+	public void tearDown() throws InterruptedException, ExecutionException {
+		languageService.dispose();
+		languageService = null;
+	}
+
+	@Test
+	public void testEmptyRelativePath() throws Exception {
+		DOMDocument document = createDOMDocument("/local-path/pom.xml", languageService);
+		List<CompletionItem> doComplete = languageService.doComplete(document, new Position(0, 14), new SharedSettings()).getItems();
+		// Some typical conventions
+		assertEquals("../pom.xml", doComplete.get(0).getLabel());
+		assertEquals("../sibling-parent/pom.xml", doComplete.get(1).getLabel());
+		assertEquals("child-parent/pom.xml", doComplete.get(2).getLabel());
+		// parent folder, then direct children
+		assertEquals("..", doComplete.get(3).getLabel());
+		assertEquals("child-parent", doComplete.get(4).getLabel());
+		assertEquals("folder1", doComplete.get(5).getLabel());
+		assertEquals("folder2", doComplete.get(6).getLabel());
+	}
+
+	@Test
+	public void testRelativePathWithPrefix() throws Exception {
+		DOMDocument document = createDOMDocument("/local-path/pom.xml", languageService);
+		List<CompletionItem> doComplete = languageService.doComplete(document, new Position(2, 17), new SharedSettings()).getItems();
+		// Some typical conventions
+		assertEquals("../pom.xml", doComplete.get(0).getLabel());
+		assertEquals("../sibling-parent/pom.xml", doComplete.get(1).getLabel());
+	}
+
+	@Test
+	public void testEmptyModulePath() throws Exception {
```

Comment:
Just a minor note: couldn't we name this `testModulePathNoPrefix` to be closer to `testModuleWithPrefix`?